Kiosk/App/BidderDetailsRetrieval.swift

```diff
@@ -39,6 +35,27 @@ extension UIViewController {
                 self.presentViewController(UIAlertController.failedBidderDetailsAlertController(), animated: true, completion: nil)
             }
     }
+
+    func emailPromptAlertController() -> UIAlertController {
+        let alertController = UIAlertController(title: "Send Bidder Details", message: "Enter your email address or phone number registered with Artsy and we will send your bidder number and PIN.", preferredStyle: .Alert)
+
+        let ok = UIAlertAction.Action("OK", style: .Default)
+        let action = CocoaAction { _ -> Observable<Void> in
+            let text = (alertController.textFields?.first)?.text ?? ""
+
+            return self
+                .retrieveBidderDetailsSignal(text)
+                .map(void)
```

Comment:
Huh good question. It's purpose is to map Observables to send Void dos satisfy the type checker, but this observable is already Void. I'll remove it, thanks!